plugins/qr-code-reader/README.md

```diff
@@ -0,0 +1,37 @@
+# QR Code Reader
+
+With this webchat plugin, you can read QR codes and send the scanned information back to the chatbot.
+In order to do so, you have to send a specifiy **SAY NODE** in Cognigy.AI which starts the plugin in your webchat:
```

Comment:
Did you mean **CODE NODE**?

Reply:
you can do it with a say node or a code node